person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly convert the `rentals` to `attr_accessor` as you already use it for writing in the Rental class initialize method.

Reply:
@gemmen29 Changed the same,